charts/golang/templates/deployment.yaml

```diff
@@ -143,34 +161,34 @@ spec:
             {{- if .Values.postgresql.enabled }}
             - name: DB_HOST
               value: {{ $dbHost }}
-            - name: DB_USER
-              value: {{ .Values.postgresql.postgresqlUsername }}
             - name: DB_DATABASE
-              value: {{ .Values.postgresql.postgresqlDatabase }}
+              value: "postgres"
+            - name: DB_USER
+              value: "postgres"
```

Comment:
Changing these to "static" values, since the user 'postgres' and the database 'postgres' will always exist by default.

Also, since, it seems, the chart does not let us define the username of the admin user any longer...

https://github.com/bitnami/charts/blob/master/bitnami/postgresql/README.md#:~:text=auth.enablePostgresUser

Reply:
The alternative is not using the admin user, which implies:

* Creating a new database
* Creating a new user
* Creating a new password for the new user

This would, in theory, be better, but is a change of behaviour...

(There are helm chart values for this).